rclpy/rclpy/time.py

```diff
@@ -41,21 +51,22 @@ def __init__(
         self._time_handle = _rclpy.rcl_time_point_t(total_nanoseconds, clock_type)
 
     @property
-    def nanoseconds(self):
+    def nanoseconds(self) -> int:
+        """:return: the total number of nanoseconds since the clock's epoch."""
```

Comment:
I see that some places we use `:return:`, and others we use `:returns:`.  Assuming they both work, can we be consistent (at least within this file)?

Reply:
Sure, though it might not matter. They [seem to be aliases](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists), and the generated documentation looks identical to me.

Reply:
b14ca2c